fullstory-segment-middleware/src/main/java/com/fullstorydev/fullstory_segment_middleware/FullStorySegmentMiddleware.java

```diff
@@ -159,4 +167,118 @@ private BasePayload getNewPayloadWithFSURL(BasePayload payload, Map<String, Obje
         }
         return null;
     }
+
+    private Map<String, Object> getSuffixedProps (Map<?, ?> props) {
```

Comment:
Nit: it looks like you can provide a more specific type for the props argument:
![image](https://user-images.githubusercontent.com/45576380/85858856-4e955200-b78a-11ea-8496-f588d6f2c4dd.png)

this from: https://github.com/segmentio/analytics-android/blob/master/analytics/src/main/java/com/segment/analytics/Properties.java

Reply:
me & Josh: same page'n

:)